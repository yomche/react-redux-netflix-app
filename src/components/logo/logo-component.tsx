import React, { FC } from 'react';
import { StyledLogo } from './logo-styles';

export const LogoComponent: FC = () => <StyledLogo />;
