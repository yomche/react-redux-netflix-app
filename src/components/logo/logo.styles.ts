import styled from 'styled-components';
import { netflixLogo } from './images';

export const StyledLogo = styled.div`
  width: 120px;
  height: 30px;
  background-image: url(${netflixLogo});
  background-size: 120px 30px;
`;
