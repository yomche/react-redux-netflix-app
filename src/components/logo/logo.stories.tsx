import React, { ComponentProps } from 'react';

import { Story } from '@storybook/react';

import { Logo } from './logo.component';

export default {
  title: 'Components/Logo',
  component: Logo,
};

const Template: Story<ComponentProps<typeof Logo>> = (args) => <Logo {...args} />;

export const Primary = Template.bind({});
Primary.args = {};
