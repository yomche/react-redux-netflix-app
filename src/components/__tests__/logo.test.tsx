import React from 'react';
import { render } from '@testing-library/react';

import { Logo } from '../logo/logo.component';

describe('< Logo /> component', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
