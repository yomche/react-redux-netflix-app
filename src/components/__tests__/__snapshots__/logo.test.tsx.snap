// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< Logo /> component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="StyledLogo-sc-1wiya5k-0 ehakiy"
  />
</DocumentFragment>
`;
